### PR TITLE
Added catify.js

### PR DESCRIPTION
**Pull request by Denton-L**
Replaces all images with pictures of cats.

### CODE REVIEW

File: src/catify.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function catify(){
```
Comment (Rabrennie) [inline src/catify.js]: should be var catify = function()

File: src/catify.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+		var imgs = document.getElementsByTagName("img");
+		for (var i = 0; i < imgs.length; i++) {
+				var img = imgs[i];
+				img.src="http://placekitten.com/"+img.clientWidth+"/"+img.clientHeight;
+		}
+}
```
Comment (Rabrennie) [inline src/catify.js:7]: missing anything.prototype.catify = catify

**Comment by Denton-L** [comment]
Updated with requested changes

**Comment by Rabrennie** [comment]
Thanks :+1:

**Pull request merged**